src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -811,7 +739,7 @@ using PosListsByChunk = std::vector<std::shared_ptr<PosLists>>;
  */
```

Comment:
I don't get what this function does from the comment. Is `column i` correct?
It's not your text, but you probably know what is done here without first digging deep into the code.

Reply:
You mean `setup_pos_lists_by_chunk`, right? No, luckily (?) I didn't really have to look into that.